docs/source/release_notes.rst

```diff
@@ -2,15 +2,22 @@
 
 Release Notes
 -------------
-Future Release
-==============
+.. Future Release
+  ==============
     * Enhancements
     * Fixes
-        * Remove 24th hour from PartOfDay primitive and add 0th hour (:pr:`2167`)
     * Changes
     * Documentation Changes
     * Testing Changes
 
+.. Thanks to the following people for contributing to this release:
+
+
+v1.11.1 Jul 5, 2022
+====================
```

Comment:
I think you have an extra `=` in this line

Reply:
removed!